server/controllers/temporary-accommodation/manage/assessmentsController.ts

```diff
@@ -40,6 +40,33 @@ export const archivedAssessmentsTableHeaders = [
     },
   },
 ]
+export const confirmationPageContent: Record<string, Record<'title' | 'text', string>> = {
```

Comment:
Can the type be more precise? Does:

`Record<Assessment['status'], {title: string, text: string}>`

work?